### ISSUE TITLE
Accessible view buttons are always enabled even if can not cycle further

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#229560]
Testing #229460

In the Github Copilot walkthrough accessible view. It is not possible to move further than 5 levels down as there are 5 steps. I can however continue pressing on the up and down arrows, as they are always enabled.

**Comment by bhavyaus** [comment]
@meganrogge is there a way for the content providers to enable/disable the button status?

**Comment by meganrogge** [comment]
no, this would be tricky to do because providers iterate through results one by one vs providing all upfront. pretty sure there's an issue open for this already, but fine with me to track this here. 

The accessibility solution for this that is implemented in some places (chat responses for ex) is to alert with the index and total count when navigating between them.

**Comment by meganrogge** [comment]
I don't think I'll get to this
